Mono.Cecil/AssemblyWriter.cs

```diff
@@ -3156,7 +3156,7 @@ static string GetXmlSecurityDeclaration (SecurityDeclaration declaration)
 
 		void WriteTypeReference (TypeReference type)
 		{
-			WriteUTF8String (TypeParser.ToParseable (type));
+			WriteUTF8String (TypeParser.ToParseable (type, false));
```

Comment:
Can you pass false as named argument here please?

Reply:
Done.